### PR TITLE
Ncnews 7

### PR DESCRIPTION
**Pull request by bathcoder**
1. Refactored /api/articles/:article_id to include comment count
2. Made new describe block to test this tiskct.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -168,3 +169,78 @@ describe("GET /api/users", () => {
...
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            comment_count: 11
+          })
+        )
+      );
+  });
+  test("status 200 : Returns the article given in the endpoint, with the username as the author and added comment count", () => {
```
Comment (emilyb93) [inline __tests__/app.test.js]: This is a test for the same behaviour as the previous test, we dont need it

File: models/articles_model.js [hunk 2]
```diff
@@ -3,7 +3,19 @@ const { convertTimestampToDate } = require("../db/helpers/utils.js");
 
 exports.fetchArticleById = (article_id) => {
   return db
-    .query("SELECT * FROM articles WHERE article_id = $1", [article_id])
+    .query("ALTER TABLE articles ADD comment_count INT")
```
Comment (emilyb93) [inline models/articles_model.js]: We do not want to add the comment as a new column, we want to be getting the comment count in a SELECT query. Also this doubles your time and queries made. 

Have a look at how to get your comment count within a SELECT statement, have a look into COUNT(), LEFT JOIN, and GROUP BY as a starting point, and be sure to check the notes app!

**Review by emilyb93 — CHANGES_REQUESTED** [review]
Please refactor your comment count query

**Review by cjplatten — APPROVED** [review]
Looks good!

**Pull request merged**